contracts/MarginCalculator.sol

```diff
@@ -72,13 +73,22 @@ contract MarginCalculator is Initializable {
         // Vault contains no short tokens: return collateral value.
         if (_isEmptyAssetArray(_vault.shortOtokens)) return (SignedConverter.intToUint(collateralAmount.value), true);
 
+        // get required margin, denominated in strike or underlying asset
         FixedPointInt256.FixedPointInt memory marginRequirement = _getMarginRequired(_vault);
```

Comment:
can we split this up for puts and calls? 

have it as marginRequiredInStrikeAsset = .... 
and marginRequiredInUnderlyingAsset = ... 

if not the naming is a little confusing

Reply:
Alternatively can all this happen in the getMarginRequired function?